security/sec-iride/src/test/java/org/geoserver/security/iride/identity/IrideIdentityTokenizerTest.java

```diff
@@ -80,7 +79,7 @@ public void testTokenizationThrowNullPointerExceptionForNullValue() throws Iride
      */
     @Test(expected = IrideIdentityMissingTokenException.class)
     public void testTokenizationFailForEmptyOrBlankValue() throws IrideIdentityMissingTokenException {
-        final String value = StringUtils.repeat(" ", RandomUtils.nextInt(10));
+        final String value = randomBlankOrEmptyString();
```

Comment:
I think the splitted tests are better, they are testing a wider range of cases and only those. With randomic behaviour each time you run the suite you are testing different stuff.